Dockerfile

```diff
@@ -1,23 +1,18 @@
 # Openhim Console Dockerfile for latest changes
-FROM ubuntu:16.04
+FROM node:14.17-alpine
+RUN apk upgrade --update-cache --available
+RUN apk add git
 
-WORKDIR /etc/
+RUN mkdir -p /app
+WORKDIR /app
 
-# Update apt-repo list and install prerequisits
-RUN apt-get update
-RUN apt-get install -y git
-RUN apt-get install -y bzip2
-RUN apt-get install -y curl
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get install -y nodejs
-
-# Clone Openhim-console repo
-RUN git clone https://github.com/jembi/openhim-console.git
-
-WORKDIR /etc/openhim-console
+COPY package.json /app
+COPY . .
 
 # Install dependencies and build
-RUN npm i
+RUN npm install
+RUN npm install webpack-dev-server && npm install webpack webpack-cli --save-dev
```

Comment:
These are dev dependencies, are they needed for the prod build?

Reply:
I see they are needed but the regular install will include them

Reply:
We should change the npm run build to npm run build prod. This one includes the dev dependencies and I do not know if it includes anything that may affect you in production